serve/opentelemetry.go

```diff
@@ -26,10 +26,15 @@ import (
 
 // newResource returns a resource describing this application.
 func newResource(p *plugin.Plugin) *resource.Resource {
+	def := resource.Default()
+	schemaURL := def.SchemaURL()
+	if schemaURL == "" {
+		schemaURL = semconv.SchemaURL
+	}
 	r, err := resource.Merge(
```

Comment:
This method always returns `r` (even with `err`, which the single value can be `ErrSchemaURLConflict`) so we could also do `if r != nil  { return r } panic(err)`